### PR TITLE
Add a Manpage overhaul project description

### PR DESCRIPTION
**Pull request by gbergling**
I have written a project description for my currently effort on improving the FreeBSD manpages.

### CODE REVIEW

File: 2020q1/manpages.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## FreeBSD Manpages overhaul ##
+
+Contact: Gordon Bergling <gbergling@gmail.com> 
+
+I am currently working on an overhaul for the FreeBSD manpages by updating the HISTORY and STANDARDS sections and while here creating new manpages for parts of the system that missing documentation. FreeBSD has already one of the best documentation available for an UNIX-like operation system, but there are parts that could be improved.
+
+For the parts that have been already improved you can have a look at my Phabricator account at https://reviews.freebsd.org/.
```
Comment (lwhsu) [inline 2020q1/manpages.md:7]: You probably want directly link to your profile: https://reviews.freebsd.org/p/gbergling_gmail.com/

Comment (gbergling) [reply 2020q1/manpages.md]: I changed the link, thank you for the hint.

**Comment by gbergling** [comment]
I added the .DS_Store file to the local .gitignore. I hope that fixes the problem.

**Comment by debdrup** [comment]
That's just included a .gitignore file in the the pull request, weird. You know what, it's not worth it to delay the pull request, I'll just git rm the files and fix it that way once I've pulled it. Thank you very much for your report, and your work! :)

**Pull request merged**